libraries/standard/mqtt/src/iot_mqtt_operation.c

```diff
@@ -131,6 +131,33 @@ static IotMqttError_t _scheduleCallback( _mqttOperation_t * pOperation );
 static bool _completePendingSend( _mqttOperation_t * pOperation,
                                   bool * pDestroyOperation );
 
+/**
+ * @brief Initialize newly created  MQTT operation.
+ *
+ * @param[in] pMqttConnection The MQTT connection associated with the operation.
+ * @param[in] pOperation pointer to the new operation.
+ * @param[in] flags Flags variable passed to a user-facing MQTT function.
+ * @param[in] pCallbackInfo User-provided callback function and parameter.
+ *
+ *
+ * @return #IOT_MQTT_SUCCESS or #IOT_MQTT_NO_MEMORY.
+ */
+static IotMqttError_t _initializeOperation( _mqttConnection_t * pMqttConnection,
+                                            _mqttOperation_t * pOperation,
+                                            uint32_t flags,
+                                            const IotMqttCallbackInfo_t * pCallbackInfo );
+
+/**
+ * @brief Send MQTT Ping Request to the broker.
+ *
+ * @param[in] pMqttConnection The MQTT connection associated with the request.
+ * @param[in] pPingreqOperation The Ping Request Operation.
+ *
+ * @return `true` if send successful or; false if not.
+ */
+static bool _sendPingRequest( _mqttConnection_t * pMqttConnection,
+                                        _mqttOperation_t * pPingreqOperation );
```

Comment:
Considering that `pPingreqOperation` = `&( pMqttConnection->pingreq )`, the second parameter seems unnecessary. Also, this line needs to be uncrustified, possibly due to tabs instead of spaces.